cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -2,30 +2,32 @@ import { TextField, Button, Select, TextArea, Modal, HTML, including, MultiColum
 import getRandomPostfix from '../../../utils/stringTools';
 
 const saveButton = Button('Save as profile & Close');
-
 const organizationModal = Modal('Select Organization');
 
 const marcBib = 'MARC Bibliographic';
-
 const incomingRecordType = {
   marcBib: 'MARC Bibliographic',
   edifact: 'EDIFACT invoice',
 };
-
 const folioRecordTypeValue = {
   instance: 'Instance',
   holdings: 'Holdings',
   item: 'Item',
   invoice: 'Invoice',
   marcBib: 'MARC Bibliographic'
 };
-
 const organization = {
   gobiLibrary: 'GOBI Library Solutions',
   harrassowitz: 'Otto Harrassowitz GmbH & Co. KG',
 };
+const holdingType = {
```

Comment:
'holdingType' is assigned a value but never used.

Reply:
I deleted it